app/presenters/valid_payments_array.rb

```diff
@@ -44,11 +44,15 @@ def choices_to_present
 
       choices.append(
         PaymentType::ONLINE
-      ) if govuk_pay_enabled?
+      ) if govuk_pay_enabled? && c100_application.online_submission?
```

Comment:
This is redundant.
Online payment was already being filtered in the `govuk_pay_enabled?` method.

Note the line:
```ruby
return false unless c100_application.online_submission?
```

Reply:
addressed